### PR TITLE
refactor(combsort): implement MutableSorter trait for CombSorter

### PR DESCRIPTION
**Pull request by borghol**
- Implementing the MutableSorter trait for CombSorter
- Standardized the tests by adding the descending test case to the macro and removing it from tests that use it

### CODE REVIEW

File: src/sorting/radix_sort.rs [hunk 1]
```diff
@@ -47,13 +47,6 @@ mod tests {
         assert_sorted!(&a);
     }
 
-    #[test]
-    fn descending() {
-        let mut v = vec![201, 127, 64, 37, 24, 4, 1];
-        radix_sort(&mut v);
-        assert_sorted!(&v);
-    }
```
Comment (alexfertel) [inline src/sorting/radix_sort.rs:55]: Why was this removed?

**Review by alexfertel — APPROVED** [review]
LGTM! Only a small question.

**Comment by borghol** [comment]
Hey, I moved the descending test to the sorting_tests macro, and you can’t have two tests with the same name. I could’ve changed the test name but that would’ve been redundant in the radix_sort case.
